Tasks/HelmDeployV0/src/utils.ts

```diff
@@ -98,4 +98,12 @@ export function addVersion(helmCli: helmcli, version: string) {
         helmCli.addArgument("--version ".concat(version));
     else
         console.log("The given version is not valid. Running the helm install command with latest version");
+}
+
+export function replaceNewlinesWithCommas(overrideValues: string): string {
+    const keyValuePairs = overrideValues.split("\n").filter((pair) => { 
```

Comment:
Can be shortened to `...filter(pair => pair)`.
Is it empty string or null?

Reply:
Will update. I'm not sure whether we need to have a check for null or empty strings. I tested with this change and it seemed to work for extra newlines scenario.

Reply:
Ohh, for me filtering null didn't work. Anyways, the current code looks harmless.